test/artists.test.ts

```diff
@@ -59,7 +56,7 @@ describe('Artist requests', () => {
                 '1WgXqy2Dd70QQOU7Ay074N',
             ]);
 
-            for (let i = 0; i < severalArtistsResponse.length; i++) {
+            for (let i = 0; i < severalArtistsResponse.length; i += 1) {
```

Comment:
:thinking: was this change necessary? i++ is better readable I guess

Reply:
The linter recommends us not to use ++.